### ISSUE TITLE
Site is Currently Down

### ISSUE THREAD

**Issue by RedTatsu256** [cashto/resistance#35]
The site is indeed down again. I hope it's back up soon.

**Comment by bilbobags** [comment]
Yes, and thank you for maintaining this site and feeding our insatiable addiction.

**Comment by cashto** [comment]
Website is up but the database seems down ... can't ping it.

I'll have to look more closer after I get off work in a few hours.

**Comment by cashto** [comment]
.... and apparently now the database is back and the site is running.

I'll keep an eye on it the next few hours.
